modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -103,7 +117,10 @@ load balancers and routers to manage the traffic.
 one IP address pool. If you need to access the services from an external network,
 configure load balancers and routers to manage the traffic.
 <10> You must set the platform to `none`. You cannot provide additional platform
-configuration variables for bare metal infrastructure.
+configuration variables for
+ifndef::ibm-z[bare metal]
```

Comment:
Does power need to be represented here, or is it okay that it says bare metal?

Reply:
Good catch, added the IBM Power conditional.